### ISSUE TITLE
"target group" for windows in Rust compiler team

### ISSUE THREAD

**Issue by nikomatsakis** [rust-windows/wg#5]
Hi folks,

Recently the compiler team's "meta working group" has been contemplating creating a "Windows target group", as you can read about in https://github.com/rust-lang/compiler-team/issues/293 (which links to [this hackmd](https://hackmd.io/TriAwtjpQ5iCs37QvUdYVg?view). 

Someone pointed me at this group, which I didn't know existed! (Or, if I did, I had forgotten.) I'm excited to find it. Let's chat! It seems like there would be a lot of overlap here, though I imagine the group I am proposing would be more narrowly focused on rustc and core toolchain issues, whereas this group perhaps has a broader focus.

**Comment by nikomatsakis** [comment]
There is a [Zulip stream](https://rust-lang.zulipchat.com/#narrow/stream/233931-t-compiler.2Fmajor-changes/topic/create.20windows.20target.20group.20compiler-team.23293) here btw associated with rust-lang/compiler-team#293.

**Comment by retep998** [comment]
I don't see why this group can't do both broader focus stuff, and narrower rustc and core toolchain issues. Some of the narrower stuff would require looking at things from a broader focus anyway, and there would be so much overlap.

Perhaps we can start with just this one group for now, pulling in anyone you think would be a good candidate, and in the future we can reevaluate whether we need to split off more focused sub teams.

**Comment by Lokathor** [comment]
Also, niko, this group mostly just declared itself to exist "unofficially" one day a few months ago. As far as I know we never really actually *communicated* our declaration back to anyone on any official team at any point. So don't feel bad that you "forgot" thus group. Quite the opposite.

**Comment by ratijas** [comment]
@retep998 
> pulling in anyone

![image](https://user-images.githubusercontent.com/6737986/82115128-95466400-9769-11ea-85cb-fe538a548f23.png)

**Comment by nikomatsakis** [comment]
> I don't see why this group can't do both broader focus stuff, and narrower rustc and core toolchain issues. 

I would like to have the "official" compiler setup that I described regardless, because it would integrate into our tooling, and allow us to do things like `@rustbot ping windows` and have everybody get cc'd. That's probably the most immediately useful thing. (I think that the idea of growing up a more active "area of expertise" that can help guide active design would be a nice byproduct, and probably something we would look to do after we've got the notification list up and going.)

Because of how github teams work, I don't think we can "ping" a team like `@rust-windows/wg` or something, though, so I think y'all would have to add your names to the list individually.
